compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TypeResolver.java

```diff
@@ -1734,7 +1734,7 @@ private BType resolveTypeDesc(BLangTableTypeNode td, ResolverData data) {
         BTableType tableType = new BTableType(TypeTags.TABLE, symTable.empty, null);
         BTypeSymbol typeSymbol = type.tsymbol;
         tableType.tsymbol = Symbols.createTypeSymbol(SymTag.TYPE, Flags.asMask(EnumSet.noneOf(Flag.class)),
-                typeSymbol.name, typeSymbol.originalName, typeSymbol.pkgID,
+                typeSymbol.name, typeSymbol.originalName, data.env.enclPkg.symbol.pkgID,
```

Comment:
```suggestion
                typeSymbol.name, typeSymbol.originalName, symEnv.enclPkg.symbol.pkgID,
```

When there are multiple usages of `data.env`, as in this case, we have extracted it to `symEnv` and used that instead. Apply this in other areas where it is applicable.

Reply:
Thank you for the helpful suggestions. Will work on these changes and let you know

Reply:
Fixed with be8c54dffd2c10afe0dab61cdf0d91f5fb702ff2. Found one more similar case in `BLangStreamType` and applied it there as well.